pkg/proc/bininfo.go

```diff
@@ -911,10 +911,11 @@ func (image *Image) Close() error {
 	return err2
 }
 
-func (image *Image) setLoadError(fmtstr string, args ...interface{}) {
+func (image *Image) setLoadError(logger *logrus.Entry, fmtstr string, args ...interface{}) {
```

Comment:
Do we never surface this error? I'm questioning the need to inject the logger at all.

Reply:
We do when we first create the target, in NewTarget, but not for additional images that appear later (like plugins/dlls). I think this is the right behavior, but we should still log it.
In the connected issue the error was *not* surfaced, but it was because we were dropping it in the for loops I've changed, not because of this.